charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -150,14 +153,14 @@ spec:
         - --cacert=/var/etcd/ssl/ca/ca.crt
         - --insecure-transport=false
         - --insecure-skip-tls-verify=false
-        - --endpoints=https://{{ .Values.name }}-local:{{ .Values.etcd.clientPort }}
+        - --endpoints=https://0.0.0.0:{{ .Values.etcd.clientPort }}
         # enable TLS on backup-restore server reusing etcd cert bundle
         - --server-cert=/var/etcd/ssl/server/tls.crt
         - --server-key=/var/etcd/ssl/server/tls.key
 {{ else }}
         - --insecure-transport=true
         - --insecure-skip-tls-verify=true
-        - --endpoints=http://{{ .Values.name }}-local:{{ .Values.etcd.clientPort }}
+        - --endpoints=http://0.0.0.0:{{ .Values.etcd.clientPort }}
```

Comment:
Please revert this. `{{ .Values.name }}-local` is resolved to `127.0.0.1`.

Reply:
Done